### PR TITLE
Adding new merchant to the BCH gang.

### PR DESCRIPTION
**Pull request by maesitos**

### CODE REVIEW

File: _data/retail.yml [hunk 1]
```diff
@@ -1080,6 +1080,20 @@ websites:
...
   othercrypto: false
+- name: Monorean
+  url: https://www.monorean.com/
+  img: monorean.png
+  twitter: monorean
+  facebook: monorean
+  bch: Yes
+  btc: No
+  othercrypto: No
+  doc: http://blog.monorean.com/en/bitcoin-cash-accepted-here
```
Comment (kenman345) [inline _data/retail.yml]: I’m on my mobile but is this able to work as HTTPS?

Also, are we sure they don’t also accept BTC? We like to be accurate about listings.

This does look like an exciting new merchant!

Comment (kenman345) [reply _data/retail.yml]: Just tested and they do not have HTtPS for the blog section. No need to change this then

**Comment by kenman345** [comment]
Can’t take all the credit for the base tasks but the Add based tasks were
all my doing. They’ve been incredibly useful for processing requests made
to our repository and in streamlining the process. The verification script
has also been updated from where we started to help notify you of mistakes.
Let us know if you have any thoughts on improvements though if you think we
can do better

**Comment by maesitos** [comment]
@kenman345 is master being deployed? As for Travis it says it is but the changes of at least the last few PR are not being displayed in production.

**Comment by kenman345** [comment]
Travis deploys to our beta site whatever is in Master. Once a week or when
we tag master branch for release, it builds a docker container that we
deploy to our live site

**Pull request merged**